scripts/build/templates/icon-object.js

```diff
@@ -2,7 +2,8 @@
   title: '%s',
   slug: '%s',
   get svg() {
-    return '<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>%s</title><path d="' + this.path + '"/></svg>';
+    d()
+    return getSvg(this)
```

Comment:
Ok, so the change is:

`'<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>%s</title><path d="' + this.path + '"/></svg>'` (125 characters) by `getSvg(this)` (12 characters), but `getSvg` seems very general as said before. Replacing with `getSvgFromSimpleIcon` would be 26 characters.

Other option is to avoid exposing functions at all and set at the top `const a = '<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>`... and adding a `htmlTitle` property, but minifying the names of the variables at the top, using for example `a` and `b`, would be something like: `a+this.htmlTitle+b+this.path+c` + `title:''` + length of the title encoded (38 characters + length of the title encoded).

Anyways, have you tested the performance decreasement of running 5 regexes for each character in the title? Maybe to reading the file is a faster process 😅 I think that the reduction in bundle size is not worth, I prefer to start with an `htmlTitle` if we want to remove the long string in the `svg` getter.

Additionally, I'm not totally against exposing an utility file, but I would prefer to expose it entirely, not only 3 functions and testing all utilities.

Reply:
Hmm ok, let's go with your change for now. I'll close this.